sale_order_line_no_print/models/sale_order.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    hide_in_report = fields.Boolean(
+        help="This line will be hidden in quotations for customers"
+    )
```

Comment:
suggestion: please [think positive when using booleans](https://www.recallstack.icu/en/2020/09/01/do-not-use-negative-booleans-please/):

```suggestion
    display_in_report = fields.Boolean(
        default=True,
        help="Display this line in quotations for customers."
    )
```

Reply:
oks, changed to fit the opposite